mythril/laser/ethereum/instructions.py

```diff
@@ -811,7 +809,7 @@ def mstore_(self, global_state):
         try:
             # Attempt to concretize value
             _bytes = util.concrete_int_to_bytes(value)
```

Comment:
Extract should be used here vs concretizing and getting bytes that way. @norhh Could you create an issue for this?

Reply:
can you explain why extract should be used? Created an issue #685

Reply:
The current process to get bytes is:
symbol -> concrete int -> bytes

Which breaks for symbolic values, in which case could do:
symbol -> [Extract(0, 7, value), ....]